### ISSUE TITLE
accessibility of microsoft vs 2015 isolated shell for visual studio 2017 community 32 bit

### ISSUE THREAD

**Issue by StarTrekCafe** [MicrosoftDocs/visualstudio-docs#2042]
hi. a blind it student and when i try to launch sql management studio 2017.9.1, and also had to install the upgrade setup for ssme, as it was falling over when it got to 44 % and the vs 2015 kb 3 million 95 681 update, and now when i was able to install the upgrade setup. now when i try to then launch from the desktop or all apps list windows 10 1809, says one or more components could not be found. please reinstall. i reckon it is a corrupted visual studio isolated 2015 shell. but i use three screen readers, jaws for windows from http://www.freedomscientific.com, non visual desktop access (nvda), http://www.nvaccess.org and windows narrator. cannot seem to get to the items to repair, or reinstall. so need a more accessible shell, and maybe have more accessible controls, and buttons, and keyboard short cuts. is there a more accessible. go and download the demo jaws 2019 and also the free nvda, and use windows narrator for windows 10, then try to use the shell with a screen reader, and you will see the issue. this needs to be repaired, but the normal programs add and replace or uinstall, does not read it at all. so need to get this shell repaired and also then able to get sql management studio then able to then launch, and then import my databases. using visual studio 2017 community 15.9.3. latest stable version.,.  

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7254052d-2d37-509e-e6c8-d5674ea775ca
* Version Independent ID: a2acc423-07d3-409c-7de1-21dd88e3cdcd
* Content: [Report a problem - Visual Studio](https://docs.microsoft.com/en-gb/visualstudio/ide/report-a-problem?view=vs-2017)
* Content Source: [docs/ide/report-a-problem.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/report-a-problem.md)
* Product: **visual-studio-dev15**
* GitHub Login: @seaniyer
* Microsoft Alias: **seiyer**

**Comment by dnethi** [comment]
Hello StarTrekCafe

Thank you for reporting the issue. We have updated the VS shell for SSMS 18 which has several Accessibility updates. Could you try the latest version of SSMS which is SSMS 18 and let us know if you are still having issues installing it.

thanks,
Dinakar

**Comment by DanteGagne** [comment]
Hey StarTrekCafe,

Thanks for your feedback. We've been making a lot of improvements specifically around accessibility in Visual Studio 2019. I think you'll find that the compatibility with screen readers is much better with VS2019 than ever before. If you have further feedback, please let us know.
